client/BUILD

```diff
@@ -56,4 +58,5 @@ assemble_maven(
 deploy_maven(
     name = "deploy-maven",
     target = ":assemble-maven",
+    deployment_properties = "@graknlabs_dependencies//distribution:deployment.properties",
 )
```

Comment:
Same here, you might want to run a tool or something to catch all missing newlines in the repo

Reply:
added